packages/scandipwa/src/component/WishlistItem/WishlistItem.component.js

```diff
@@ -70,7 +71,7 @@ export class WishlistItem extends PureComponent {
                   defaultValue: description
               } }
               events={ {
-                  onChange: changeDescription
+                  onChange: ({ target: { value } = {} }) => changeDescription(value)
```

Comment:
What do u think if we will handle this issue on level of Field component? I mean instead of getting value from target everywhere, we can make it to take in 1 function and then pass value to onChange custom function we setting.

Reply:
Or I guess it can be as it is and just in another task we can handle it.

Reply:
It is doable in `<Field/>` component (I wrote such a logic), yet the problem is that a great variety of `onChange` handlers with different parameters is used throughout the app, which is why it is hard to unify this logic. I left it as is at least for now.